### DISCUSSION TITLE
http.efi unable to download file

### DISCUSSION THREAD

**Discussion by ccy** [tianocore/edk2#3313] [General]
When I use `http.efi` like:

    http -i eth4 google.com index.html

I received this message:

    Unable to download the file 'index.html' on 'eth4'  - Access Denied

The interface `eth4` is an working interface.  It works for `tftp.efi`

**Comment by lzap** [comment]
In SecureBoot turned on, it is not possible to use `HTTP` you have to use `HTTPS` is I believe what you have here.
